pkg/skaffold/schema/latest/config.go

```diff
@@ -146,6 +148,24 @@ type TestCase struct {
 	StructureTests []string `yaml:"structureTests,omitempty"`
 }
 
+// Deployer contains properties shared between deployers
+type Deployer struct {
```

Comment:
I find it a bit troubling that this type is defined in `latest/config.go`. This means that whenever a new api version is introduced, this logic (and the tests) need to be duplicated. What about moving this type into `package util` (`schema/util`)? The only drawback I see here is that the schema definition is no longer completely self-contained per api version.
@nkubala what do you think?

Reply:
Not in schema/util but in pkg/runner, schema package should only be responsible the actual config schema operations

:YAML parsing, default values, profiles, config upgrade